extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1987,18 +1990,27 @@ describes.realWin('amp-analytics', {
       config['resourceTimingSpec'] =
           config['triggers'][0]['resourceTimingSpec'];
       delete config['triggers'][0]['resourceTimingSpec'];
-      runResourceTimingTest(
-          [entry], newConfig(), 'https://ping.example.com/endpoint?rt=');
+      return runResourceTimingTest(
+          [entry], config, 'https://ping.example.com/endpoint?rt=');
     });
 
-    it('should only report timings on ini-load', () => {
+    it('should only report timings only once', () => {
       const entry = newPerformanceResourceTiming(
           'http://foo.example.com/lib.js?v=123', 'script', 100, 500, 10 * 1000,
           false);
+      // Stub performance.now so that it returns a timestamp after the resource
+      // timing entry.
+      sandbox.stub(win.performance, 'now').returns(700);
       const config = newConfig();
-      config['triggers'][0]['on'] = 'visible';
-      runResourceTimingTest(
-          [entry], config, 'https://ping.example.com/endpoint?rt=');
-    });
+      config['triggers'][0]['on'] = 'timer';
+      config['triggers'][0]['timerSpec'] = {'interval': 1};
+      return runResourceTimingTest(
+          [entry], config, 'https://ping.example.com/endpoint?rt=' +
+              'foo_bar-script-100-500-7200', 3500 /* timeout */)
+          .then(() => {
+            expect(sendRequestSpy.args[1][0]).to.equal(
+                'https://ping.example.com/endpoint?rt=');
+          });
+    }).timeout(4000);
```

Comment:
What does this timeout do? With the timeout can we still be sure that `expect(sendRequestSpy.args[1][0]).to.equal('https://ping.example.com/endpoint?rt=');` has been tested?